nominations/models.py

```diff
@@ -365,7 +512,9 @@ def save(self, *args, **kwargs):
         if self.status == 'submitted' and self.submitted_dt is None:
```

Comment:
Isn't this a duplicate save method?

Reply:
Aw man something messed up in merge with master, will fix.